modules/Microsoft.Batch/batchAccounts/.test/app.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
```

Comment:
We may have an issue with adding more than 3 parallel tests for batch accounts given the default quota per subscription 1-3 ( Ref https://learn.microsoft.com/en-us/azure/batch/batch-quota-limit#resource-quotas).

Given that app and pool are child resources, would it be possible to add them both to the main parameters.json test?

That would be the preferred approach anyway for consistency with other modules, trying to test as many parameters as possible in the parameters.json. If that wasn't possible, I'd suggest to use different locations than the default west europe for the new tests.